src/guiguts/mainwindow.py

```diff
@@ -534,6 +562,42 @@ def handle_configure(self, _e: tk.Event) -> None:
             except tk.TclError:
                 pass
 
+    def next_image(self, reverse: bool = False) -> None:
+        """Load the next image alphabetically.
+
+        Args:
+            reverse: True to load previous image instead.
+        """
+        if not self.filename:
+            sound_bell()
+            return
+
+        # Check current directory is valid
+        current_dir = os.path.dirname(self.filename)
+        if not os.path.isdir(current_dir):
+            logger.error(f"Image directory invalid: {current_dir}")
+            return
+
+        # Turn off auto-image; we're going to manual control.
+        if root().auto_image_state.get():
```

Comment:
I think it would be slightly preferable to replace these two lines with
`preferences.set(PrefKey.AUTO_IMAGE, False)`
i.e. to turn the Pref off, rather than directly manipulate the variable (although both should work). There are other places where auto-image gets turned off using the Pref.

Reply:
Ok, done